platform/ios/export/export_plugin.cpp

```diff
@@ -433,6 +444,26 @@ void EditorExportPlatformIOS::_fix_config_file(const Ref<EditorExportPreset> &p_
 		} else if (lines[i].contains("$export_method")) {
 			int export_method = p_preset->get(p_debug ? "application/export_method_debug" : "application/export_method_release");
 			strnew += lines[i].replace("$export_method", export_method_string[export_method]) + "\n";
+		} else if (lines[i].contains("$provisioning_profile_specifier_debug")) {
+			String specifier = provisioning_profile_specifier_dbg_variant.get_type() != Variant::NIL ? provisioning_profile_specifier_dbg_variant : "";
+			strnew += lines[i].replace("$provisioning_profile_specifier_debug", specifier) + "\n";
+			if (valid_dbg_specifier == false) {
+				add_message(EXPORT_MESSAGE_WARNING, TTR("Export"), vformat("application/provisioning_profile_specifier_debug not found in export_presets"));
+			}
```

Comment:
Thanks for looking at the changes, but has_valid_export_configuration seem to only be returning if the configuration is valid. 

Its valid to not have "application/provisioning_profile_specifier_debug" or "application/provisioning_profile_specifier_release" since the change is backwards compatible, but it does generate a warning if the project file expects those values to be replaced.  I don't want to double parse the configuration file to generate that warning, so I'd prefer to keep it where it is.

Alternatively I could make those fields required and return them as errors if they are not in the export templates, if you really want it in has_valid_export_configuration.

Reply:
When config is valid `has_valid_export_configuration` can return `true`, but add a warning string to `r_error`. In this case it will be displayed in yellow at the bottom of export dialog, and it's already done for similar options, like disabled notarization on macOS for example.

Reply:
I ended up not being able to repro the error case I was trying to warn about, so I've removed my warnings entirely.